README.md

```diff
@@ -447,10 +447,10 @@ if `$promisesOrValues` contains 0 items.
 $promise = React\Promise\some(array $promisesOrValues, integer $howMany);
 ```
 
-Returns a promise that will resolve when `$howMany` of the supplied items in
+Returns a promise that will resolve when at least `$howMany` of the supplied items in
```

Comment:
*at least* is wrong here. The output promise fulfills after exactly `$howMany`fulfill and ignores the rest.

Reply:
I've replaced word *resolve* with *fulfill*.

My intention with word **at least** was to show, that it is enough for `$howMany` promises to be fulfilled. For example, when we have 3 pending promises, it is enough for 2 of them to be fulfilled. Not exactly 2 should be fulfilled, and 1 should fail.